src/CoreBundle/private/library/classes/dbobjects/TCMSTableConf.class.php

```diff
@@ -181,7 +190,21 @@ public function &GetFields(&$oTableRow, $loadDefaults = false, $bDoNotUseAutoObj
                     if (array_key_exists($oField->name.'-original', $oTableRow->sqlData)) {
                         $oField->data = $oTableRow->sqlData[$oField->name.'-original'];
                     } else {
-                        $oField->data = $oTableRow->sqlData[$oField->name];
+                        // Standard case
+                        // Try to find the correct field name for the data in $oTableRow with respect to the language
+
+                        $fieldName = $oField->name;
+
+                        if ($language !== null) {
+                            // TODO GetEditFieldNameForLanguage() returns wrong if the base translation language === current language
```

Comment:
"wrong" --> "false"?

Is this comment really needed?

Reply:
Removed

Couldn't reproduce a problem here just now.